index.html

```diff
@@ -80,32 +80,137 @@ <h1 class="header1">Hey there. I'm &nbsp;&nbsp;&nbsp;&nbsp;Mariane&nbsp;&nbsp;&n
+            </div>
+        </div>
+        <div class="workp2">
+             <div class="work-frame">
+                <h3 class="work-title">Multi - Post Stories Gain+Glory</h3>
+                <ul class="lang">
+                    <li class="lang-list-item">Ruby on rails</li>
+                    <li class="lang-list-item">css</li>
+                    <li class="lang-list-item">Javascript</li>
+                    <li class="lang-list-item">html</li>
+                </ul>
+                <button type="submit" class="buttonW">See Project</button>
+            </div>
+        </div>
+        <div class="workp3">
+             <div class="work-frame">
+                <h3 class="work-title">Multi - Post Stories Gain+Glory</h3>
+                <ul class="lang">
+                    <li class="lang-list-item">Ruby on rails</li>
+                    <li class="lang-list-item">css</li>
+                    <li class="lang-list-item">Javascript</li>
+                    <li class="lang-list-item">html</li>
+                </ul>
+                <button type="submit" class="buttonW">See Project</button>
+            </div>
+        </div>
+        <div class="workp4">
+             <div class="work-frame">
+                <h3 class="work-title">Multi - Post Stories Gain+Glory</h3>
+                <ul class="lang">
+                    <li class="lang-list-item">Ruby on rails</li>
+                    <li class="lang-list-item">css</li>
+                    <li class="lang-list-item">Javascript</li>
+                    <li class="lang-list-item">html</li>
+                </ul>
+                <button type="submit" class="buttonW">See Project</button>
+            </div>
+        </div>
+        <div class="workp5">
+             <div class="work-frame">
+                <h3 class="work-title">Multi - Post Stories Gain+Glory</h3>
+                <ul class="lang">
+                    <li class="lang-list-item">Ruby on rails</li>
+                    <li class="lang-list-item">css</li>
+                    <li class="lang-list-item">Javascript</li>
+                    <li class="lang-list-item">html</li>
+                </ul>
+                <button type="submit" class="buttonW">See Project</button>
+            </div>
+        </div>
+        <div class="workp6">
+             <div class="work-frame">
+                <h3 class="work-title">Multi - Post Stories Gain+Glory</h3>
+                <ul class="lang">
+                    <li class="lang-list-item">Ruby on rails</li>
+                    <li class="lang-list-item">css</li>
+                    <li class="lang-list-item">Javascript</li>
+                    <li class="lang-list-item">html</li>
+                </ul>
+                <button type="submit" class="buttonW">See Project</button>
+            </div>
+        </div>
+            <!-- About Section
+    <section class="about-sect"> -->
+        <div class="about">
```

Comment:
- Please, kindly use a section tag for your about section and it shouldn't be wrapped inside the works section as this is another section. please fix this.

Reply:
*FONT SIZE*

- Please, kindly improve the font size, and weight of **all your text** as this is making your web page hard to read its content as it doesn't improve the user experience.